examples/threading/src/main.rs

```diff
@@ -5,12 +5,16 @@
 
 use riot_rs::debug::println;
 
+const STACK_SIZE_THREAD1: usize = 4096;
```

Comment:
Would you remove the changes here? Let's have a proper PR introducing defaults and using them.